closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -2354,14 +2362,138 @@ Java_jdk_crypto_jniprovider_NativeCrypto_ECCreatePrivateKey
 
     (*env)->ReleasePrimitiveArrayCritical(env, s, nativeS, JNI_ABORT);
 
+    if (NULL != sBN) {
+        (*OSSL_BN_free)(sBN);
+    }
+
     if (0 == ret) {
         return -1;
     }
 
     return 0;
 }
 
-/* Encode an EC Elliptic Curve over a Prime Field
+/* Encode an EC Elliptic Curve over a Prime Field */
+EC_KEY* ECEncodeGFp(BIGNUM *aBN,
+                    BIGNUM *bBN,
+                    BIGNUM *pBN,
+                    BIGNUM *xBN,
+                    BIGNUM *yBN,
+                    BIGNUM *nBN,
+                    BIGNUM *hBN)
```

Comment:
This function (and `ECEncodeGF2m()`) should be static with the name at the start of a new line:
```c
static EC_KEY *
ECEncodeGFp(BIGNUM *aBN,
            BIGNUM *bBN,
            BIGNUM *pBN,
            BIGNUM *xBN,
            BIGNUM *yBN,
            BIGNUM *nBN,
            BIGNUM *hBN)
{
```
with a space before `*` consistently.

Reply:
This update ( and the same type of update to `ECEncodeGF2m` ) is complete.